games/c.yaml

```diff
@@ -616,6 +616,33 @@
   type: remake
   updated: 2019-10-26
 
+- name: Children of Solarus
+  originals:
+  - Legend of Zelda - A Link to the Past
+  type: similar
```

Comment:
```suggestion
  type: remake
```

Reply:
I think Mystery of Solarus DX should be a remake too, as it uses the same assets, and the game play is the same, even though the story and levels are different
For this game, since the only similarity with the original game is the gameplay, it should be a clone instead